### ISSUE TITLE
Timecodes for "ML Zoomcamp 8.10 - Data Augmentation"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#151]
Youtube video: https://www.youtube.com/watch?v=aoPfVsS3BDE

**Comment by dimzachar** [comment]
0:00:00 - Creating more data from existing images
0:01:30 - Flipping and mirroring image transformations explained.
0:03:00 - Explaining image transformations: flipping, rotating, filling
0:04:38 - Rotating and shifting images in Keras.
0:06:25 - Image transformations: shift, shear, zoom.
0:08:14 - Shrinking and zooming images in Keras.
0:09:59 - Data augmentation with image manipulation techniques.
0:11:38 - Data augmentation parameters for image generator.
0:13:22 - Zoom, flip, and shear image augmentation.
0:15:07 - Symmetry, rotation, and augmentation for models.
0:16:43 - Experimenting with augmentations for better accuracy
0:18:25 - Training neural network with data augmentation.
0:20:26 - Optimizing GPU and CPU usage during training.
0:22:16 - Troubleshooting poor validation accuracy in TensorFlow.
0:24:46 - Tuning neural networks for reproducibility.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
